docs/general/how-to/use-token-creator.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Use the Token Creator
+sidebar_position: 2
+---
+
+# Use the Token Creator
+
+<div class="video-container">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/wveXdqB0pNk?si=M7bEFMMEfwxtu8uR" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></div>
+
+This guide explains how to create tokens and NFTs using the **Token Creator** on [**Universal Everything**](https://universaleverything.io/).  
+It covers how to set up, choose token types, add metadata, and manage created tokens.
+
+## Overview
+
+The **Token Creator** allows you to:
+
+- Create and deploy tokens or NFTs without writing code
+- Define categories, metadata, and co-creators
+- Manage and update tokens directly from your **Universal Profile**
+
+This process provides a straightforward way to issue on-chain assets for events, collectibles, memberships, and digital items.
+
+## Accessing the Token Creator Tool
+
+![Connect wallet and select Token Creator](../../../static/img/general/how-to/token-creator/create-asset.png)
+
+1. Go to **[Universal Everything](https://universaleverything.io/)**.
+2. Connect your **Universal Profile** and visit your profile.
+3. On the Tokens tab, select **Create Asset**.
+
+## Creating a Token
+
+Follow these steps to create a new token:
+
+### 1. Enter Token Details
+
+Fill in the basic information for your token:
+
+| Field                   | Description                                                          |
+| ----------------------- | -------------------------------------------------------------------- |
+| **Name\***              | The display name of your token (e.g., MyCustomToken, Fan Token)      |
```

Comment:
```suggestion
| **Name\***              | The display name of your token (e.g., Fan Token)      |
```

Why is there an escape character `\*`?

Also suggest better token name example (the first one looks like a `contract` name)